tests/e2e/e2e_common.go

```diff
@@ -329,3 +333,8 @@ func setClusterFlavor(clusterFlavor cnstypes.CnsClusterFlavor) {
 		rwxAccessMode = true
 	}
 }
+
+func setPodImage(varName string) {
+	podimage := os.Getenv(varName)
+	os.Setenv(busyBoxImageOnGcr, podimage)
```

Comment:
why are we setting env var here, we can just set `busyBoxImageOnGcr` variable right

Reply:
code is modified now